src/node_perf.cc

```diff
@@ -261,6 +261,30 @@ void LoopIdleTime(const FunctionCallbackInfo<Value>& args) {
   args.GetReturnValue().Set(1.0 * idle_time / NANOS_PER_MILLIS);
 }
 
+void UvMetricsInfo(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  uv_metrics_t metrics;
+
+  // uv_metrics_info always return 0
+  uv_metrics_info(env->event_loop(), &metrics);
```

Comment:
It's better to make the assertion in code.

```suggestion
  CHECK_EQ(uv_metrics_info(env->event_loop(), &metrics), 0);
```